samples/subsys/shell/sensor_module/src/main.c

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2018 Diego Sueiro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(app);
+
+void main(void)
+{
+
```

Comment:
is this sample supposed to be doing something? or is main() blank because we just interact with the shell prompt?

Reply:
The main() blank because we just interact with the shell prompt. Should I get rid of this?